Build/Projects/Framework.Content.Pipeline.References.definition

```diff
@@ -43,6 +43,7 @@
     <Binary
       Name="ATI.TextureConverter"
       Path="ThirdParty\Dependencies\ATI.TextureConverter\ATI.TextureConverter.dll" />
+    <Reference Include="MonoGame.Framework.Core" />
```

Comment:
You don't need to do this here, you can just add `<Reference Include="MonoGame.Framework.Core" />` outside all of the platform-specific blocks.